docs/integrations.md

```diff
@@ -74,6 +74,45 @@ semgrep:
     - python -m semgrep_agent --publish-deployment $SEMGREP_DEPLOYMENT_ID --publish-token $SEMGREP_APP_TOKEN
 ```
 
+</p>
+</details>
+<details><summary>Circle CI</summary>
+<p>
+
+```yaml
+version: 2.1
+jobs:
+  semgrep-scan:
+    parameters:
+      repo_path:
+        type: string
+        default: myorg/semgrep-test-repo
+      default_branch:
+        type: string
+        default: main
+      semgrep_deployment_id:
+        type: integer
+        default: << my deployment id  >>
```

Comment:
Sorry, don't mean to be nitpicky, but wouldn't want a new user to get confused. The `<<` and `>>` delimiters have specific meaning in Circle CI, which aren't needed here. I like that you included delimeters, and think they help readability. I'm thinking:

```
default: *my deployment id*
```